### PR TITLE
chore(docs): faq - pain points wing solves

### PR DESCRIPTION
**Pull request by ShaiBer**
Fixes: #2182

*By submitting this pull request, I confirm that my contribution is made under the terms of the [Monada Contribution License](https://docs.winglang.io/terms-and-policies/contribution-license.html)*.

### CODE REVIEW

File: docs/07-faq/006-what-pains-wing-solves.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+3. **Cloud vendor lock-in**: Reliance on a single cloud provider can limit flexibility and increase risk.
+4. **Need for multi-cloud**: Organizations require the ability to deploy and manage applications across multiple cloud providers.
+
+With Wing, the goal is to improve the developer experience by treating the entire cloud as its target computer and bringing the benefits of smart compilers that handle the lower levels of the stack to cloud development. This approach helps developers focus on creating value for users, be more independent, and enjoy instant feedback with faster iteration cycles, making cloud application development more enjoyable and efficient.
+
+Developers should be able to write code with a "pencil" rather than a "pen," iterating quickly and learning from their mistakes. With Wing, developers can make a change and test it within milliseconds, even in an airplane with no WiFi, just like in the 90s. By addressing these pain points, Wing aims to revolutionize cloud development and create a better developer experience.
+
+You can experience Wing's benefits with zero friction in our [playground](https://play.winglang.io).
```
[the remark this answers is not in the document]
Comment (ShaiBer) [reply docs/07-faq/006-what-pains-wing-solves.md:27]: We're talking about the pains we solve, the main one being the development experience.
Then we talk a bit about how we solve them, so it made sense to me to add a way for devs to experience our improved DX with zero friction

[the remark this answers is not in the document]
Comment (ShaiBer) [reply docs/07-faq/006-what-pains-wing-solves.md]: I made the connection clearer now

File: docs/07-faq/006-what-pains-wing-solves.md [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+
+1. **Slow iteration cycles**: Long deployment times and difficulty testing and debugging cloud code hinder productivity and take developers out of their creative flow.
+2. **Too many tools**: Numerous tools solve small problems but don't address the root cause of difficulties in cloud development.
+3. **Lack of local development solutions**: There's a need for full simulation, visualization, and fast reloading in local development environments.
+4. **Complex cloud mechanics**: Developers need to understand cloud mechanics, such as IAM policies and networking, to be effective, even though these mechanics don't have a direct impact on the value they create for users.
+
+**Organization pain points:**
+
+1. **Dev/ops friction**: Poor separation of concerns between development and operations teams can cause tension.
+2. **Error-prone security/observability**: Ensuring security and observability can be challenging and prone to mistakes.
```
[the remark this answers is not in the document]
Comment (ShaiBer) [reply docs/07-faq/006-what-pains-wing-solves.md]: Added now

File: docs/07-faq/006-what-pains-wing-solves.md [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+3. **Cloud vendor lock-in**: Reliance on a single cloud provider can limit flexibility and increase risk.
+4. **Need for multi-cloud**: Organizations require the ability to deploy and manage applications across multiple cloud providers.
+
+With Wing, the goal is to improve the developer experience by treating the entire cloud as its target computer and bringing the benefits of smart compilers that handle the lower levels of the stack to cloud development. This approach helps developers focus on creating value for users, be more independent, and enjoy instant feedback with faster iteration cycles, making cloud application development more enjoyable and efficient.
+
+Developers should be able to write code with a "pencil" rather than a "pen," iterating quickly and learning from their mistakes. With Wing, developers can make a change and test it within milliseconds, even in an airplane with no WiFi, just like in the 90s. By addressing these pain points, Wing aims to revolutionize cloud development and create a better developer experience.
```
[the remark this answers is not in the document]
Comment (ShaiBer) [reply docs/07-faq/006-what-pains-wing-solves.md:25]: Tweaked the voice here now

**Review by shaiber01 — APPROVED** [review]

**Pull request merged**